sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3752,11 +3752,38 @@ piEnqueueKernelLaunch(pi_queue Queue, pi_kernel Kernel, pi_uint32 WorkDim,
   PI_ASSERT((WorkDim > 0) && (WorkDim < 4), PI_INVALID_WORK_DIMENSION);
 
   if (GlobalWorkOffset != NULL) {
-    for (pi_uint32 i = 0; i < WorkDim; i++) {
-      if (GlobalWorkOffset[i] != 0) {
-        return PI_INVALID_VALUE;
+    uint32_t Count = 0;
+    ZE_CALL(zeDriverGetExtensionProperties,
+            (Queue->Device->Platform->ZeDriver, &Count, nullptr));
+
+    if (Count == 0) {
+      zePrint("No extensions supported on this driver\n");
```

Comment:
```suggestion
      zePrint("No global offset extension found on this driver\n");
```

I think it is important to say that **global offset** is not supported

Reply:
ok.

Reply:
this is for the case where count = 0, not checking for any specific ext.